### DISCUSSION TITLE
the shortcut key for change file  in nvim

### DISCUSSION THREAD

**Discussion by longintlong** [ayamir/nvimdots#1040] [Q&A] [answered]
As shown below, i want to ask Is there any shortcut key for switching files?
<img width="621" alt="image" src="https://github.com/ayamir/nvimdots/assets/39492651/a1cb8179-9880-4481-8712-2937f286e37e">

btw, If i jump to the implementation of a function and want to return to the cursor's previous position, is there a corresponding shortcut key for this?

Thx!

**Comment by kang8** [comment]
> there any shortcut key for switching files

I don't know if nvimdots has a shortcut key, but I map `<D-]>` and `<D-[>` to quick move buffer

```lua
vim.keymap.set('n', '<D-]>', '<cmd>bnext<cr>')
vim.keymap.set('n', '<D-[>', '<cmd>bprevious<cr>')
```

> If i jump to the implementation of a function and want to return to the cursor's previous position

use [CTRL-O](https://github.com/neovim/neovim/blob/f2fc44550fbe5b7ebfedc2b155dc41e93f49aedb/runtime/doc/motion.txt#L1035-L1037)

**Comment by ayamir** [comment, marked as answer]
1. https://github.com/ayamir/nvimdots/blob/370bd64c703ec83ce808d9e35c18b3c831573f84/lua/keymap/ui.lua#L9-L10
2. `<C-o>` i.e. [Ctrl-o](https://github.com/neovim/neovim/blob/f2fc44550fbe5b7ebfedc2b155dc41e93f49aedb/runtime/doc/motion.txt#L1035-L1037)

**Reply by longintlong** [reply]
thanks for ur prompt reply!
It seems that many shortcut keys are not mentioned in [wiki](https://github.com/ayamir/nvimdots/wiki/Keybindings), where should I find them?

**Reply by ayamir** [reply]
Use `<C-p>` to open VSCode-like command panel and search.
